servers/sanat/roles/mediawiki/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
 - name: Install LocalSettings.php
   template: src=LocalSettings.php.j2 dest=/srv/mediawiki/workdir/LocalSettings.php
 
+- name: Copy Composer installer
```

Comment:
Composer is already installed by the `geerlingguy.composer` role in `site.yml`. Why is this required?